styles/style.css

```diff
@@ -92,11 +97,30 @@ p {
   border-left-color: var(--orange-dark);
 }
 
-
 /* Buttons */
+.btn {
+  background-color: var(--orange-dark);
+  color: #fff;
+  /* list-style-type: none; */
+  border: none;
+  border-radius: 3px;
+  padding: 5px;
+  cursor: pointer;
+}
 
-  /* INSERT BUTTON STYLES HERE */
-
+.btn:hover {
+  background-color: #f84e04;
+}
+.par-button button:last-child {
+  color: var(--orange-dark);
+  background-color: #fff;
+  border: none;
+}
+.par-button button:last-child:hover {
+  color: #f84e04;
+  background-color: rgb(243, 232, 232);
+}
+/* INSERT BUTTON STYLES HERE */
```

Comment:
Generally very good styling. The instructions specify that the hover colours should be inverted on hover, i.e. orange turns white, white turns orange.